backend/core/src/listings/tests/listings.service.spec.ts

```diff
@@ -170,6 +172,16 @@ describe("ListingsService", () => {
             getJurisdiction: jest.fn(),
           },
         },
+        {
+          provide: EmailService,
```

Comment:
I think it would be useful to add some tests to this file to test the new functionality in listings.service. Especially around `updateAndNotify`. If need be those could be e2e tests instead

Reply:
Curious how big of a lift this request may be, if smallish definitely happy to see that here, if there's a more comprehensive need for coverage would it be possible to consider adding a separate ticket to look at all test coverage at a high level? (Just for the sake of getting this feature into Doorway by EOD tomorrow)

Reply:
I don't know how much effort it would be to create these tests, but the framework is already set up so theoretically it wouldn't be hard. I'm ok if you want to split it out as a separate ticket. But hopefully it could be a quick follow up. 

Also, just curious if you want to get it in Doorway by EOD tomorrow in order to demo it on Tuesday? Or is there some other reason?

Reply:
I believe the demo is the motivation and given that, I'm going to create a separate ticket so this testing approach is not rushed! I will work on it once the demo is in a good spot so as early as tomorrow and as late as Tuesday afternoon!